data/json/npcs/campus/generic_class_definitions.json

```
@@ -0,0 +1,76 @@
+[
+  {
+    "type": "item_group",
+    "id": "FREE_MERCH_GUARD_wield_phase1",
```

```suggestion
    "id": "CAMPUS_GUARD_wield_phase1",
```
The ID should probably be changed to match the faction, and that ID could cause conflict problems with Free Merchant NPCs.